### PR TITLE
Collapse and Expand courses

### PR DESCRIPTION
**Pull request by JasonBaoZ**
screen shot 2017-03-11 at 4 21 52 pm
screen shot 2017-03-11 at 4 21 46 pm

@lbkchen @amillman Created collapse and expand courses button

### CODE REVIEW

**Comment by amillman** [comment]
looks good

File: app/assets/javascripts/courseEdit/components/CourseEditPage.js [hunk 1]
```diff
@@ -133,6 +135,10 @@ class CourseEditPage extends React.Component {
     this.setCourseValue('isPublished', !this.state.course.isPublished)
   }
 
+  toggleIsCollapsed() {
+    this.setState({ forceOpen: !this.state.forceOpen})
```
Comment (lbkchen) [inline app/assets/javascripts/courseEdit/components/CourseEditPage.js]: nit: Add space before closing brace

File: app/assets/javascripts/courseEdit/components/CourseEditPage.js [hunk 2]
```diff
@@ -176,11 +182,15 @@ class CourseEditPage extends React.Component {
...
+  renderCollapsedLabel() {
+    return this.state.forceOpen ? 'Collapse Courses' : 'Expand Courses'
+  }
+
   renderSections() {
     return this.state.course.sections.map((value) => {
       return (
         <div className='component-edit-section' key={value.id}>
-          <SectionEdit section={value} deleteSection={this.deleteSection} />
+          <SectionEdit section={value} deleteSection={this.deleteSection} forceOpen={this.state.forceOpen}/>
```
Comment (lbkchen) [inline app/assets/javascripts/courseEdit/components/CourseEditPage.js]: nit: add space before closing slash

File: app/assets/javascripts/courseEdit/components/CourseEditPage.js [hunk 3]
```diff
@@ -244,7 +254,12 @@ class CourseEditPage extends React.Component {
             >
               {this.renderPublishLabel()}
             </button>
-
+            <button
+            className='button marginLeft-sm'
+              onClick = {this.toggleIsCollapsed}
+            >
```
Comment (lbkchen) [inline app/assets/javascripts/courseEdit/components/CourseEditPage.js]: nit: props and className need to be indented onto next level

**Comment by lbkchen** [comment]
👍

**Pull request merged**